src/main/java/org/apache/maven/plugin/patch/ApplyMojo.java

```diff
@@ -237,6 +238,16 @@
     @Parameter( defaultValue = "false" )
     private boolean binary;
 
+    /**
+     * List of ANT-style exclude patterns. These patterns are used to exclude patch files found
+     * in the {@link ApplyMojo#patchDirectory}. These exclude patterns does not apply to a static defined list of
+     * {@link ApplyMojo#patches}.
+     *
+     * @since 1.3
+     */
+    @Parameter
+    private List excludes;
```

Comment:
```suggestion
    private List<String> excludes;
```

Reply:
AFAIK this plugin was written in Java 1.4, not a lot of maintenance so nobody cared about generics. However we should, especially when adding new code.

Reply:
Ok thanks for explaining, I changed it.